modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -4611,6 +4611,24 @@ else if (log.isInfoEnabled())
             log.info("\"" + registry + "\" not found.");
     }
 
+    /** {@inheritDoc} */
+    @Override public void configureHitRateMetric(String name, long rateTimeInterval) {
+        try {
+            ctx.metric().configureHitRate(name, rateTimeInterval);
+        } catch (IgniteCheckedException e) {
+            throw new IgniteException(e);
+        }
+    }
+
+    /** {@inheritDoc} */
+    @Override public void configureHistogramMetric(String name, long[] bounds) {
```

Comment:
I really don't like that `IgniteKernal` interface contains this method. It isn't "kernal" functionality at all.

Reply:
Agree. Let's create separate JMX-bean for metrics configuration purposes?

Reply:
I'll introduce bean in IGNITE-11927 for metrics enabling/disabling. So it could be good place for it. I also offer to consider possibility introduce such method for every bean and for every histogram that will be created for registry.

Reply:
I moved metrics management methods to the `MetricsMxBean`